### PR TITLE
[16.0][MIG] purchase_order_product_recommendation: Migration to version 16.0

### PR DESCRIPTION
**Pull request by Yadier-Tecnativa**
@Tecnativa TT42423

### CODE REVIEW

**Comment by Yadier-Tecnativa** [comment]
ping @sergio-teruel and @pedrobaeza

**Review by pedrobaeza — APPROVED** [review]

File: purchase_order_product_recommendation/wizards/purchase_order_recommendation_view.xml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+                        <group name="col_right">
+                            <field
+                                name="show_all_partner_products"
+                                attrs="{'invisible': [('show_all_products', '!=', False)]}"
+                            />
+                            <field name="show_all_products" />
+                        </group>
+                    </group>
+                    <group>
+                        <field name="line_ids" nolabel="1">
```
Comment (pedrobaeza) [inline purchase_order_product_recommendation/wizards/purchase_order_recommendation_view.xml]: This is not taking the whole width of the wizard. Try `col=2` or similar.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Review by Yadier-Tecnativa — COMMENTED** [review]
Change the width to fill all the space.

File: purchase_order_product_recommendation/wizards/purchase_order_recommendation_view.xml [hunk 2]
```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="purchase_order_recommendation_view_form" model="ir.ui.view">
+        <field name="name">Recommended Products for this Customer</field>
```
Comment (pedrobaeza) [inline purchase_order_product_recommendation/wizards/purchase_order_recommendation_view.xml]:
```suggestion
        <field name="name">Recommended Products for this Vendor</field>
```

**Review by pedrobaeza — APPROVED** [review]

**Review by pilarvargas-tecnativa — APPROVED** [review]
LGTM :)

File: purchase_order_product_recommendation/wizards/purchase_order_recommendation_view.xml (hunk 2 above)
Comment (Yadier-Tecnativa) [reply purchase_order_product_recommendation/wizards/purchase_order_recommendation_view.xml]: Done

**Comment by sergio-teruel** [comment]
ping @juangomezzurita

**Pull request merged**